crates/cheatcodes/src/test.rs

```diff
@@ -100,3 +118,32 @@ fn breakpoint(state: &mut Cheatcodes, caller: &Address, s: &str, add: bool) -> R
 
     Ok(Default::default())
 }
+
+/// Gets chain information for the given alias.
+fn get_chain(state: &mut Cheatcodes, chain_alias: &str) -> Result {
+    // Parse the chain alias - works for both chain names and IDs
+    let alloy_chain = AlloyChain::from_str(chain_alias)
+        .map_err(|_| fmt_err!("invalid chain alias: {chain_alias}"))?;
+
+    // Check if this is an unknown chain ID by comparing the name to the chain ID
+    // When a numeric ID is passed for an unknown chain, alloy_chain.to_string() will return the ID
+    // So if they match, it's likely an unknown chain ID
+    if alloy_chain.to_string() == alloy_chain.id().to_string() {
+        return Err(fmt_err!("invalid chain alias: {chain_alias}"));
+    }
```

Comment:
ah I see, we're actually missing a helper function for this on the chaintype, so this is a workaround